src/logic/wallets/getWeb3.ts

```diff
@@ -113,3 +113,8 @@ export const getSDKWeb3Adapter = (signerAddress: string): Web3Adapter => {
     signerAddress,
   })
 }
+
+export const getSafeSDK = async (signerAddress: string, safeAddress: string): Promise<Safe> => {
```

Comment:
If the signerAddress is always `connectedWalletAddress`, you need not send it as an argument but instead select it here.